src/lib.rs

```diff
@@ -1027,6 +1027,7 @@ fn run_test(
         &mut errors,
     );
     cmd.args(&extra_args);
+    cmd.env("RUSTC_UITEST_TEST_RUN", "True");
```

Comment:
I think we can leave off the `RUSTC` prefix, this crate can run a lot of other programs, too